SU-SWS/saa_victoria_subtheme
# Adapt 357  banner options

# READY FOR REVIEW

# Summary
- Add theme function to add has-image or doesn't have image classes for banners
- Add styles so banners can look differently:
top banner with image, top banner without image, top banner image only, inside the page banner with image, inside the page banner without an image

# Review By (Date)
- When does this need to be reviewed by?
6/8/2020

# Urgency
- merging quickly will allow me to continue if tomorrow they confirm to continue with this approach.

# Steps to Test

1. pull down branch, 
1. create pages with banner types: top banner with image, top banner without image, top banner image only, inside the page banner with image, inside the page banner without an image
2. Then preview and see different styles exist and are started. Styles are not fine-tuned, just basic "outline" of styles.

# Associated Issues and/or People
- ADAPT-357 exploring banner options
- ADAPT-356 Main top banner for image & text

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
